### PR TITLE
Patch for Infinium Human Methylation BeadChip Training

### PR DESCRIPTION
**Pull request by kpbioteam**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,13 +0,0 @@
-source "https://rubygems.org"
```
Comment (bgruening) [inline Gemfile]: please do not remove those files

Comment (kpbioteam) [reply Gemfile]: Hi Bjorn, I used to have them in my gitignore. Sorry for the inconvenience I re-added them now

File: topics/epigenetics/tutorials/ewas-suite/slides.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
 ## Overview
 
 - Applications of EWAS
-- The Data (450k what is it?)
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/slides.html]: @kpbioteam on top can you add your name as well and `questions:` etc as metadata?

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 3]
```diff
@@ -1,38 +1,50 @@
...
 
-title: "EWAS data analysis of 450k data"
+title: "Infinium Human Methylation BeadChip"
 zenodo_link: "https://zenodo.org/record/1251211#.WwREQ1Mvz-Y"
 questions:
-  - "What is a epigenome-wide association studies?"
-  - "Why is a EWAS analysis useful?"
-  - "Why 450k data?"
+  - "Which DNA regions and positions are diffrentialy methylated in pre MAPKi treatment and post MAPKi resistance Melanomas GSE65183?"
+  - "How to analyse and visualise Infinium Human Methylation BeadChip's?"
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md]: is it really just one hour? :)

Comment (kpbioteam) [reply topics/epigenetics/tutorials/ewas-suite/tutorial.md]: yes, as it is a small subset of the original data

File: topics/epigenetics/tutorials/ewas-suite/slides.html (hunk 2 above)
Comment (kpbioteam) [reply topics/epigenetics/tutorials/ewas-suite/slides.html]: I have updated name and add metadata.yaml

**Comment by bgruening** [comment]
Thanks @kpbioteam!

**Pull request merged**